webcompat/static/js/lib/issue-list.js

```diff
@@ -336,14 +336,18 @@ issueList.IssueView = Backbone.View.extend(
       // There are some params in the URL
       if (urlParams.length !== 0) {
         queryMatch = urlParams.match(this._searchRegex);
+        if (queryMatch) {
```

Comment:
I wonder if there's a difference in behavior for logged-in or not logged-in users...